src/components/DropDown.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+export default function DropDown({ type, dropdown, filteredAnimals }) {
+  const handleChange = (e) => {
+    filteredAnimals(e.target.value);
+  };
+
+  return (
+    <select value={type} onChange={handleChange}>
+      {dropdown.map((animal) => (
+        <option key={animal}>{animal}</option>
+      ))}
```

Comment:
Consider hard coding an 'all' option so the user doesn't expect to see the first item in the drop down on page load.